src/control/cmd/dmg/pool.go

```diff
@@ -137,7 +137,7 @@ func (sf *sizeFlag) UnmarshalFlag(fv string) (err error) {
 
 	sf.bytes, err = humanize.ParseBytes(fv)
 	if err != nil {
-		return errors.Wrapf(err, "invalid size %q", fv)
+		return errors.Errorf("invalid size %q", fv)
```

Comment:
Why do you remove the `Wrapf()` which seems OK for me.

Reply:
+1

Reply:
same as above.